DataProcessing/datax-udf-samples/pom.xml

```diff
@@ -51,7 +51,7 @@ SOFTWARE
 
     <groupId>com.microsoft.datax</groupId>
     <artifactId>datax-udf-samples_2.4_2.11</artifactId>
-    <version>1.2.8-SNAPSHOT</version>
+    <version>1.2.9-SNAPSHOT</version>
```

Comment:
Also change the dependency version of datax-core_2.4_2.11 to 1.2.9-SNAPSHOT on line 116

Reply:
updated that